public/locales/az/gamepage.json

```diff
@@ -32,9 +32,9 @@
             "title": "Quraşdırmanı dayandırın"
         },
         "uninstall": {
-            "checkbox": "Siz də prefiksi silmək istərdinizmi? Bu geri qaytarıla bilməz.",
-            "checkbox_prefix": "Prefiks",
             "message": "Bu oyunu silmək istəyirsiniz?",
+            "prefixcheckbox": "Remove prefix: {{prefix}}{{newLine}}Note: This can't be undone and will also remove not backed up save files.",
```

Comment:
the problem with the approach of changing the translation key is that we will lose all translated ones, we should keep as it is and then just add the new one. Even though the current name is not specific.

OR you can change the translations files to use the new key as well, it will give conflicts with weblate though.